critiquebrainz/frontend/views/review.py

```diff
@@ -211,8 +214,10 @@ def create():
             return redirect(url_for('user.reviews', user_id=current_user.id))
 
         is_draft = form.state.data == 'draft'
+        if not form.text.data:
```

Comment:
TextAreaField in wtforms returns an empty string when no input is provided. So, it must be changed to None before inserted into the database.